app/routes/overview/IndexRoute.js

```diff
@@ -19,7 +19,7 @@ import { selectPinnedPolls } from 'app/reducers/polls';
 import { votePoll } from 'app/actions/PollActions';
 
 const mapStateToProps = (state) => ({
-  loadingFrontpage: state.frontpage.fetching,
+  loadingFrontpage: !state.frontpage.fetching,
```

Comment:
Why do you do this?

Reply:
Followed example from EventDetailRoute.js
https://github.com/webkom/lego-webapp/blob/0417f452d48ffebbb5f21e019702b7c9bdcb0e40/app/routes/events/EventDetailRoute.js#L116

Reply:
In `eventDetailRoute` the prop is opposite from here, i.e. `notLoading` is `true` whenever we are **not** loading :upside_down_face:. That's why it's negated.

Reply:
Also, if `loadingFrontpage` is not used in the `Overview` component anymore, you should remove it.